lib/engine/step/bankrupt.rb

```diff
@@ -39,10 +39,14 @@ def process_bankrupt(action)
 
         player.shares_by_corporation.each do |corporation, _|
           next unless corporation.share_price # if a corporation has not parred
-          next unless (bundle = @game.sellable_bundles(player, corporation).max_by(&:price))
 
-          @game.sell_shares_and_change_price(bundle)
-          @round.recalculate_order
+          # Do a potential repeated sell of bundles. This is important for NdM in 18MEX
+          # which might have 5% bundle(s) besides the 10%+ bundles.
+          # Most other titles this will just sell one, the largest, bundle.
+          while (bundle = @game.sellable_bundles(player, corporation).max_by(&:price))
+            @game.sell_shares_and_change_price(bundle)
+            @round.recalculate_order
```

Comment:
Recalculate order only needs doing once

Reply:
Correct.
Before it was once per corporation that could be sold. 
Have now moved it out of that loop so it is just called once in all - should work.